config/webpack/webpack.config.js

```diff
@@ -198,19 +202,19 @@ const buildWebpackConfigs = builds.map(
           rules: [
             {
               test: /\.js$/,
-              exclude: thirdPartyModulesRegex,
+              include: internalJs,
```

Comment:
The reason for the change was I didn't want to make a dynamic regex so I moved to an absolute path approach. Personally I feel it's much cleaner add style guide isn't Harford throughout so much.